### ISSUE TITLE
About training code and the resutls

### ISSUE THREAD

**Issue by holyhao** [Wangt-CN/MTFN-RR-PyTorch-Code#3]
Tanks for your work. I training the model on f30k with the default setting. But the loss dose not convergence and keep at about 51, and the test accuary is really poor. Do you have some idea about this.

**Comment by xiaoyuan1996** [comment]
me too, loss doesn't converge. Maybe the reason of triplet loss? or some problems about data preprocessing?

**Comment by Wangt-CN** [comment]
Hi, sorry for the delay replay since I haven't seen the issue.
Actually, the triplet loss is hard to converge and the parameters are very important and sensible. 
The loss 51 is indeed the bottleneck, which means you need better parameters. And if the loss can break 51. It will have a fast converge to about 20.

However, I remember the code can be good converge since I have finetune the parameters. Do you directly run the code? Or do you make any changes? I will check the code recently. Thanks for your attention to our work~

**Comment by Wangt-CN** [comment]
Hi, I have checked the results and training. It seems normal in my case. And I also just find that the previous user can implement the results with this code base [issue#1](https://github.com/Wangt-CN/MTFN-RR-PyTorch-Code/issues/1#issuecomment-523317060)
